docs/packaging.md

```diff
@@ -0,0 +1,290 @@
+# Packaging a scie executable
+
+In order to use the `scie-jump` as your application launcher, you need to 1st package it together
+with your application code and support binaries into a single file scie executable. Packaging can be
+accomplished with the `cat` command line tool alone, but you can also use the `scie-jump` itself to
+make this a bit easier. In either case you'll first need the following:
+
+1. A copy of the scie-jump executable built for the platform your scie executable will be targeting.
+2. All the files you want to package in your application.
+3. A lift manifest describing one or more commands the scie executable will be able to execute using
+   those files.
+
+There is probably a pre-built `scie-jump` binary for the platform you're targeting in our
+[releases](https://github.com/a-scie/jump/releases) that you can download. If not, you can clone
+this project on a machine of your target platform and run `cargo run --release -p package dist` and
+a `scie-jump` binary for that platform will be deposited in the `dist/` directory. See the
+[contributing guide](../CONTRIBUTING.md) for more on the development environment setup if you're not
+already setup for Rust development.
+
```

Comment:
I'd probably stick a one-liner in here to make it clear that the code below is an example use case/workflow, which will clear up this being a JVM-specific tool.

Something to the effect of: "...that varies! For Python we use XYZ, for Node we use ABC, ..."

Reply:
Done.